internal/buildscripts/packaging/choco/splunk-otel-collector/splunk-otel-collector.nuspec

```
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>splunk-otel-collector</id>
+    <version>0.0.0</version>
+    <description>
+The splunk-otel-collector is a metric-based agent written in Go that is used to monitor infrastructure and application services from a variety of environments.
+
+The splunk-otel-collector looks for a configuration file at `\ProgramData\Splunk\OpenTelemetry Collector\otelcol.exe`.  If this file does not already exist during installation, a default config file will be copied into place by the installer.

```

Is this the correct file to check for?
```suggestion
The splunk-otel-collector looks for a configuration file at `\ProgramData\Splunk\OpenTelemetry Collector\agent_config.yaml`.  If this file does not already exist during installation, a default config file will be copied into place by the installer.
```